build/crds/sedna.io_federatedlearningjobs.yaml

```diff
@@ -6622,12 +6623,21 @@ spec:
                 - model
                 - template
                 type: object
+              pretrainedModel:
+                description: TrainModel defines model of CR
```

Comment:
TrainModel is inconsistent with pretrainedModel

Reply:
fix it